sdk/storage/azure-storage-file/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Release History
+
+## 12.0.0-preview.2 (2019-08-08)
+Version 12.0.0-preview.2 is a preview of our efforts in creating a client library that is developer-friendly, idiomatic to the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide our efforts can be found in the [Azure SDK Design Guidelines for Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
+
+For details on the Azure SDK for Java (July 2019 Preview) release, you can refer to the [release announcement](https://aka.ms/azure-sdk-preview1-java).
```

Comment:
it feels like this line and line 6 in the next changelog.md should go under their own` ## 12.0.0-preview.1 (2019-07-09)` heading

Reply:
I'd better to leave a #TODO comments. I did not find the corresponding link for preview 2, will update it once it gets updated. If no specific to preview 2, we might need to keep the preview 1 release note.
Queue and file do not have preview.1. Confirmed with @rickle-msft  we can start fresh without include any previous release version. 
@g2vinay do you have any suggestion here?

Reply:
You can use https://aka.ms/azure-sdk-preview2-java in the doc

/cc @loarabia to confirm

Reply:
Yes. I'm creating the 4 language links now. That is the right pattern.

Reply:
Thanks! Done with the update.